examples/demo_interface.pp

```diff
@@ -42,14 +63,14 @@
     channel_group   => 200,
```

Comment:
You don't appear to be on the latest refpoint. The channel_group property was moved out of cisco_interface already:

https://github.com/cisco/cisco-network-puppet-module/blob/release_1.2.0/examples/demo_interface.pp#L41